test/config-next/ra.json

```diff
@@ -124,7 +124,13 @@
 						"bad-key-revoker.boulder",
 						"ocsp-responder.boulder",
 						"wfe.boulder",
-						"sfe.boulder"
+						"sfe.boulder",
+						"ca.boulder"
```

Comment:
Is this diff necessary? I think it's sufficient to add `ca.boulder` as a client of the new service, without adding it as a client of the existing service too.